advancedTodo/src/redux/todoAppComplete/todoApp.ui.js

```diff
@@ -77,100 +78,100 @@ class TodoApp extends Component {
 	}
 
 	createNotification(type) {
-    switch (type) {
-      case 'success':
-        NotificationManager.success('Information successfully saved', 'Congratulations !');
-        break;
-      case 'deleteSuccessful':
-        NotificationManager.success('Todo Deleted Successfully', 'Congratulations !');
-        break;
-      case 'completeSuccessful':
-        NotificationManager.success('Todo Completed Successfully', 'Congratulations !');
-        break;
-      case 'error':
-        NotificationManager.error('Please add information to save', 'Warning', 5000, () => {
-          alert('Error');
-        });
-        break;
-			default :
-		   	return undefined
-    }
-  }
-
- handleSearch(e){
- 	let {newTodos} = this.props;
- 	let currentList = newTodos.filter( el => (el.name).includes(e.target.value));
- 	this.setState({
- 		currentList
- 	})
- }
-
- onFocus(e){
- 	this.setState({
- 		searchActive: true
- 	})
- }
-
- handleKeyPress(e){
- 	if(e.key === 'Enter'){
- 		this.handleOnClick()
- 	}
- }
+		switch (type) {
+			case 'success':
+				NotificationManager.success(SUCCESS_MESSAGE);
+				break;
+			case 'deleteSuccessful':
+				NotificationManager.success(DELETE_MESSAGE);
+				break;
+			case 'completeSuccessful':
+				NotificationManager.success(TODO_COMPLETED_MESSAGE);
+				break;
+			case 'error':
+				NotificationManager.error('Please add information to save', 'Warning', 5000, () => {
+					alert('Error');
+				});
+				break;
+			default:
+				return undefined
+		}
+	}
+
+	handleSearch(e) {
+		let { newTodos } = this.props;
+		var currentList = newTodos.filter(el => (el.name).includes(e.target.value));
+		this.setState({
+			currentList
+		})
+	}
+
+	onFocus(e) {
+		this.setState({
+			searchActive: true
+		})
+	}
+
+	handleKeyPress(e) {
+		if (e.key === 'Enter') {
+			this.handleOnClick()
+		}
+	}
 
 	render() {

```

Comment:
This is a great opportunity to create a separate component for the Todo item. This will improve readability and maintainability as the logic for each todo item grows.